### ISSUE TITLE
Addition of more loss functions and layers in Keras-cv

### ISSUE THREAD

**Issue by soma2000-lang** [keras-team/keras-cv#1436]
I want to add more loss functions in this repository ,Any suggestions as to which loss function can be added would be very much appreciated!

cc @jbischof @mattdangerw @fchollet @LukeWood @bhack @ianstenbit @tanzhenyu

**Comment by innat** [comment]
https://github.com/keras-team/keras-cv/issues/546
https://github.com/keras-team/keras-cv/issues/330
https://github.com/keras-team/keras-cv/issues/296
https://github.com/keras-team/keras-cv/pull/1050 cc. @DavidLandup0 

However, better to pick something from [this](https://github.com/keras-team/keras-cv/issues?q=is%3Aissue+is%3Aopen+label%3Acontribution-welcome).
